mne/time_frequency/csd.py

```diff
@@ -1257,12 +1258,15 @@ def _csd_multitaper(X, sfreq, n_times, window_fun, eigvals, freq_mask, n_fft,
         Length of the FFT.
     adaptive : bool
         Use adaptive weights to combine the tapered spectra into PSD.
+    max_iter : int
+        Maximum number of iterations for weight computation (requires
+        adaptive=True).
```

Comment:
we should use the fill_doc decorator here to avoid dupes of docstrings

Reply:
Even in private functions?